pkg/sloop/webserver/webfiles_test.go

```diff
@@ -0,0 +1,49 @@
+package webserver
+
+import (
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path"
+	"testing"
+)
+
+const (
+	defaultFileMode = os.FileMode(0755)
+	someContents1   = "contents abcd"
+	filePath        = "webfiles/index.html"
+	errorString		= "Webfile %v is invalid.  Must start with %v"
```

Comment:
2 things:
1 - the fragile aspect of this is that this string has to match the one used in `readWebFiles()` - can we share the same named constant
2 - the automation runs make on this, I think the linter will catch and fail on this statement

Reply:
my mistake - sam/sam repo does the lint check on the code, sloop does not